### ISSUE TITLE
Audio button touch target is too small

### ISSUE THREAD

**Issue by BenHenning** [oppia/oppia-android#382]
The exploration player audio button's touch target is 24dp, but should be 48dp per accessibility & usability recommendations.

**Comment by nikitamarysolomanpvt** [comment]
Closed via #467

**Comment by nikitamarysolomanpvt** [comment]
Closed via #467
